.devcontainer/Dockerfile

```
@@ -1,8 +1,8 @@
-FROM ghcr.io/soham2560/humble-garden:latest
+FROM ghcr.io/soham2560/humble:latest
 
 # Add the following labels
-LABEL org.opencontainers.image.description="ROS2 Humble + ROS2 Garden + ROS2 Control"
-LABEL org.opencontainers.image.title="humble-garden-control"
+LABEL org.opencontainers.image.description="ROS2 Humble + ROS2 Harmonic + ROS2 Control"
+LABEL org.opencontainers.image.title="ROS2_TUTORIAL"
```

```suggestion
LABEL org.opencontainers.image.title="ros2-tutorial"
```
I prefer to go lower case here